docs/binary/quickstart.rst

```diff
@@ -0,0 +1,125 @@
+Quickstart Guide
+================
+
+The primary input language for cvc5 is
+`SMT-LIB v2 <http://smtlib.cs.uiowa.edu/language.shtml>`_.
+We give a short explanation of the SMT-LIB v2 equivalent of the quickstart
```

Comment:
```suggestion
We give a short explanation of the SMT-LIB v2 quickstart
```

I think that the smtlib quickstart example stands on its own. I expect many of the readers of this guide will not be aware of the corresponding cpp example.